linux_docker_resources/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN if test ${PLATFORM} = x86; then curl --silent https://packagecloud.io/gpg.ke
 RUN apt-get update && apt-get install -y build-essential ccache cmake pkg-config python3-empy python3-setuptools python3-vcstool
 
 # Install build and test dependencies of ROS 2 packages.
-RUN apt-get update && apt-get install -y clang-format-3.8 cppcheck git pydocstyle pyflakes python3-coverage python3-mock python3-nose python3-pep8 uncrustify
+RUN apt-get update && apt-get install -y clang-format-3.8 cppcheck git python3-coverage python3-mock python3-nose uncrustify
```

Comment:
they are being installed through pip in [this script](https://github.com/ros2/ci/blob/140bd1e0e150800754b9f2483e53a9f7a52921ef/ros2_batch_job/__main__.py#L49). installing them here is what is causing version conflicts I believe (see http://ci.ros2.org/view/All/job/ci_linux/1802/testReport/junit/(root)/projectroot/flake8/).